module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,15 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::vector<std::string> removeVowels(std::vector<std::string>& vec) {
+    std::vector<char> vowelVec{'a', 'e', 'i', 'o', 'u', 'y'};
+
+    for (std::string& str : vec) {
+        for (const char& vowel : vowelVec) {
```

Comment:
Reference is 4 or 8 bytes. Char is only 1
```suggestion
        for (const char vowel : vowelVec) {
```